pkg/vm/vm.go

```diff
@@ -80,6 +80,9 @@ type VM struct {
 	SyscallHandler func(v *VM, id uint32) error
 
 	trigger trigger.Type
+
+	// Invocations is a script invocation counter.
+	Invocations map[util.Uint160]int
```

Comment:
Not sure that this is the right place for `Invocations`, but we have to move it out of interop context to somewhere VM-related, because `(*PointerCallback).LoadContext` also needs to count invocations, but don't have interop context.

Reply:
`LoadContext` can accept `*interop.Context` now, after VM is inside it.

Reply:
There's one more argument for moving `Invocations` to VM: we have to increment `Invocations` each time we call `(VM).Call` outside of `vm` package. That's why I united `(VM).Call` and `v.Invocations[ctx.ScriptHash()]++`.